ediag.go

```diff
@@ -73,8 +86,16 @@ func NewEventDiagCmd() *cli.Command {
 				}
 
 				// Get the human time from the idtimestamp committed on the event.
+				idTimestamp, err := appEntry.IDTimestamp(&cmd.massif)
```

Comment:
why does this one need the massif?

Reply:
it gets the idtimestamp from the log, instead of from the event itself.

We can get it from the event if we feel its needed? but the source of truth for the idtimestamp is the log?